controller/users.js

```diff
@@ -96,8 +97,9 @@ class UserController {
         errors: { password: 'please provide your password' },
       });
     }
+
     const user = await User.findOne(
-      { where: { email, password } }
+      { where: { email } }
```

Comment:
Hey man, is there a reason why this line needed to change, I'm not sure I understand. Mind explaining, please.

Reply:
I was intending to check and compare the password in a middleware... but from the feedback I got I have to  implement the feature directly in the user's model